src/style/shared/select.less

```diff
@@ -42,3 +42,9 @@
     border-top: 1px solid @warm-gray;
     margin: 0 0.4rem;
 }
+
+.select__option__info {
+    margin-left: 1rem;
+    color: @warm-gray;
+    font-size: 12px;
```

Comment:
Although there are some exceptions, in general we should never style UI with pixel values. Instead, we want to always use relative units in order to make the UI completely scalable. (Read this for a concise overview: http://snook.ca/archives/html_and_css/font-size-with-rem). Instead, we want something like this: https://github.com/adobe-photoshop/spaces-design/blob/955e1995d80c81c0c8591f3dcbb8146303f7b219/src/style/sections/pages/face.less#L104-L106